### PR TITLE
add extra functions

### PR DESCRIPTION
**Pull request by drc38**

### CODE REVIEW

**Review by hardbyte — COMMENTED** [review]
Finally got to trying this out. Thanks!

I'll edit the API a bit and create a new PR

File: evnex/api.py [hunk 1]
```diff
@@ -324,3 +324,55 @@ async def stop_charge_point(
...
+        r = await self.httpx_client.post(
+            f"https://client-api.evnex.io/v3/charge-points/{charge_point_id}/commands/change-availability",
+            headers=self._common_headers,
+            # 'Connection': 'Keep-Alive'
+            json={"connectorId": connector_id, "changeAvailabilityType": type},
+            timeout=timeout,
+        )
+        json_data = await self._check_api_response(r)
+
+        return EvnexCommandResponse.parse_obj(json_data["data"])
```
[the remark this answers is not in the document]
Comment (drc38) [reply evnex/api.py]: What response do you get back from the v3 request?

[the remark this answers is not in the document]
Comment (drc38) [reply evnex/api.py]: image

[the remark this answers is not in the document]
Comment (drc38) [reply evnex/api.py]: Maybe it has to do with Org Tier as well? I am on 1 what are you on?

[the remark this answers is not in the document]
Comment (hardbyte) [reply evnex/api.py]: Ah I should have dug deeper. The error I get shows the "0" connectorId is the issue:

```
{"status":"400","title":"ValidateError","detail":"min 1: 0","source":{"pointer":"/connectorId"}
```


I reached out to Evnex about the org tiers, doesn't seem related:

> Org tiers are more about billing than access, though there is an Enterprise level org tier which has bespoke API access. User account roles are what determines what you can access. Most accounts are “Drivers” who are permitted to use a subset of the endpoints to view and update chargers for which they have permission. This is granted by “inviting” them to locations.

[the remark this answers is not in the document]
Comment (hardbyte) [reply evnex/api.py]: Updated to v3 and released version `0.3.1`

**Pull request merged**